frontend/apps/cloudstorage/syncservice.lua

```diff
@@ -132,11 +136,15 @@ function SyncService.sync(server, file_path, sync_cb, is_silent)
     local code_response = 412 -- If-Match header failed
     local etag
     local api = server.type == "dropbox" and require("apps/cloudstorage/dropboxapi") or require("apps/cloudstorage/webdavapi")
+    local token = server.password
+    if server.type == "dropbox" then
+        token = api:getAccessToken(server.password, server.address) or server.password -- in case using long-term token
```

Comment:
To understand that we use the long-lived token we can just check:
`self.address == nil or self.address == ""`
and not to ask Dropbox and get the error.
https://github.com/koreader/koreader/blob/fa9f0acf5e58dcc4f68d4effbdaefa795c6b67c4/frontend/apps/cloudstorage/cloudstorage.lua#L135-L148

Reply:
Thanks for pointing this out. Done now.